x/liquidity/types/utils.go

```diff
@@ -9,8 +9,7 @@ import (
 	"github.com/tendermint/tendermint/crypto"
 )
 
-// Get denom pair alphabetical ordered
-// [NOTE] WILL BE DEPRECATED in v2
+// AlphabeticalDenomPair returns denom pairs which are alphabetically sorted.
```

Comment:
```suggestion
// AlphabeticalDenomPair returns denom pairs that are alphabetically sorted.
```

Reply:
in almost all cases, we want "that" instead of "which"
(using "which" is very popular in our docs and other content, and most often is used incorrectly) see https://developers.google.com/style/pronouns 
(isn't this fun?!)